scenarios/platform.benchmarks.yml

```diff
@@ -17,8 +17,10 @@ jobs:
     readyStateText: Application started.
     framework: net5.0
     environmentVariables:
+      DOTNET_SYSTEM_NET_SOCKETS_INLINE_COMPLETIONS: 1
```

Comment:
It's fine for testing in a PR, but we don't want to run this benchmark by default with this setting. the idea is that we'll add more scenarios that represent the absolute best tuning just for numbers. We want to track the perf of the system without such tuning to detect changes that impact perf. But some discussions I had with Shay showed that we still need to have some benchmarks that represent exactly what we have or will have on TE.
Another example is that this one doesn't have PGO, but we do have some scenarios with only PGO and no inlining, or just inlining and no PGO. TE's repository has both.